content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,98 @@
+
+Let's look at the Sickest Seven, and determine next actions for improvement.  
+From the initial July release activity, the _Sickest Seven_ (7 products that took more than 5 business days over the 2 or 7 day target to publish), plus the _two outliers_ from the respins:
+
+| Platform | Version | Days over Target | Underlying Reasons / Notes | Actions to Improve | 
+| -- | -- | -- | -- | -- |
+| x64 Mac | JDK 8 | 14 | Owners need to treat as urgent, onboarding difficulties, slow connectivity | Clarify requirements with owners and improve access to TCK machines |
+| x64 Mac | JDK 18 | 27 | Owners need to treat as urgent, onboarding difficulties, slow connectivity | Clarify requirements with owners and improved access to TCK machines |
+| aarch64 Mac | JDK 17 | 6 | Bottlenecked on having only 1 machine for TCK | 2nd machine now added to temurin-compliance project |
+| aarch64 Mac | JDK 11 | 6 | Bottlenecked on having only 1 machine for TCK | 2nd machine now added to temurin-compliance project |
+| arm32 Linux | JDK 8 | 11 | Unclear, possibly slow to add an owner to it, also short-handed, team vacations | Close follow-up to ensure ownership early |
+| x86 Solaris | JDK 8 | 10 | Delayed on packaging issue | Issue now fixed, [temurin-build/3061](https://github.com/adoptium/temurin-build/issues/3061) |
+| sparcv9 Solaris | JDK 8 | 20 | Considered a best effort platform, team would/should prioritize this last | No specific action required, Reduction in effort on other platforms, and no respins would mean these lower priority platforms get released quicker |
+
+What is encouraging about this list is that most of these issues have actionable responses, many of them completed during the release, as evident in our improved times for the respins.
+
+## New Features Need New Tests
+A project that is changing, is a project that is going to occasionally break.  This is will always be the case, but how do we pro-actively protect ourselves in the tug-of-war between adding new features and remaining stable and secure?  The answer is of course 'testing'.  While we have an ever-evolving suite of tests for verifying the OpenJDK binaries we produce via the [AQAvit project](https://adoptium.net/aqavit), we additionally need to be better at adding tests to verify the changes we are making to our own build and distribution scripts.
+
+Several changes went in ahead of the July CPU that introduced delays during the release.  2 of the changes are enhancements as part of our Secure Software Development Framework [(SSDF) activities](https://github.com/adoptium/adoptium/issues/120), [GPG signing](https://blog.adoptium.net/2022/07/gpg-signed-releases/) and the [addition of SBOM artifacts](https://github.com/adoptium/temurin-build/issues/2900). Both of these features introduce new artifacts that can be downloaded alongside of the JDK binaries.  
```

Comment:
Adjust this sentence - it has a list of SSDF, GPG and SBOM then refers to "both" of these features when it looks like three. Maybe 
```suggestion
Several changes went in ahead of the July CPU that introduced delays during the release.  2 of the changes are enhancements as part of our Secure Software Development Framework [(SSDF) activities](https://github.com/adoptium/adoptium/issues/120), [GPG signing](https://blog.adoptium.net/2022/07/gpg-signed-releases/) and the [addition of SBOM artifacts](https://github.com/adoptium/temurin-build/issues/2900). The latter two of these features introduce new artifacts that can be downloaded alongside of the JDK binaries.  
```